package-lock.json

```diff
@@ -9,6 +9,21 @@
       "resolved": "https://registry.npmjs.org/@ampproject/animations/-/animations-0.2.2.tgz",
       "integrity": "sha512-Ukegl+czEVP2S6Xd48d8vR/yL1bXd2A2ccUDztW29vVOC7XrJjKSa3bIqecYckuUI+mBfJtzHJKzN6rkscz3jw=="
     },
+    "@ampproject/bento-compiler": {
+      "version": "0.0.2",
+      "resolved": "https://registry.npmjs.org/@ampproject/bento-compiler/-/bento-compiler-0.0.2.tgz",
+      "integrity": "sha512-QQn/asy6hGcR9eLNcyx55bxqgigFgu8Sh/9zEeOtR9B9Ye5WJeEx3f0vCSXaoL8Beg2v6zG3fs5flv/KuOLVXw==",
+      "requires": {
+        "@ampproject/worker-dom": "0.29.3"
```

Comment:
I see that `bento-compiler` is using `worker-dom` v0.29.3, but the core runtime dependency is v0.30.1. Could this introduce any unexpected code behavior?

Reply:
No sir...its totally fine. They can use different versions of the library without any issue